src/main/resources/language_zh-cn.properties

```diff
@@ -0,0 +1,113 @@
+add=加
+adding.new.password=加新的密码
+appName=保留密码
```

Comment:
Hi, please make changes to appName value, as suggested in the guide for translators!